gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -509,33 +509,13 @@ gst_tensor_filter_check_throttling_delay (GstBaseTransform * trans,
 }
 
 /**
- * @brief non-ip transform. required vmethod of GstBaseTransform.
+ * @brief Check input paramters for gst_tensor_filter_transform ();
  */
 static GstFlowReturn
-gst_tensor_filter_transform (GstBaseTransform * trans,
-    GstBuffer * inbuf, GstBuffer * outbuf)
+_gst_tensor_filter_transform_validate (GstTensorFilterPrivate * priv,
```

Comment:
IMO, passing the params - priv, prop and self - is unnecessary.

```
  GstTensorFilter *self = GST_TENSOR_FILTER_CAST (trans);
  GstTensorFilterPrivate *priv = &self->priv;
  GstTensorFilterProperties *prop = &priv->prop;
```

Reply:
Applied.

Reply:
Applied.